proto/extensions.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
```

Comment:
do we also need to publish this file like the rest of the proto files?

https://github.com/momentohq/client-protos/blob/main/javascript/generate_protos.sh#L5

Reply:
yeah, I expected to encounter more of these lists to bite me tbh.

Reply:
I hoped that the include path would address it but :-/ oh well

Reply:
https://github.com/momentohq/client-protos/actions/runs/4623819429/jobs/8178018886?pr=152#step:7:31

nope, this is an issue with the descriptor proto generated code.